### ISSUE TITLE
d3-geo practice

### ISSUE THREAD

**Issue by borntofrappe** [borntofrappe/learning-d3#9]
Practice with the d3-geo module and the projection function to draw a map of France and points corresponding to the villages listed on [Les Plus Beaux Villages de France](https://www.les-plus-beaux-villages-de-france.org/fr/)

**Comment by borntofrappe** [comment]
Add the project to the first table, devoted to the D3 library, and stress the different steps involved in the project, from creating the GeoJSON (or TopoJSON) file with France's coordinates to mapping the country and the villages.

**Comment by borntofrappe** [comment]
![Les Plus Beaux Villages de France](https://user-images.githubusercontent.com/33316703/196541842-c7626e2d-a854-4e53-87b6-aa299df77074.jpg)


- [x] downloads geographical data from natural earth
- [x] creates geoJSON from mapshaper.org
- [x] projects data in an `<svg>` element of known size
- [x] transitions the data as it is loaded _and_ the number of villages alongside the circles
- [x] uses the projection to highlight the villages
- [x] allows to zoom in and out
- [ ] (almost) shows the names as the map is zoomed in and there is more space for the labels

I need to document the transition and zooming features, but I'm already proud of the demo so far

**Comment by borntofrappe** [comment]
**Update**: I reviewed, refactored and restructured the project. See short clip which follows.

https://user-images.githubusercontent.com/33316703/196796644-2d61e5a8-b635-4b57-a78f-352ad9c286e4.mp4

I'm particularly proud of the way I manage the transitions, the flow of the elements. In terms of interaction the zooming feature has been replaced by a voronoi projection. By default you see a village at random. Hover on the voronoi cells to update the reference to the selected village.

Updating `README.md` closes the issue 👍
